crates/re_renderer/src/wgpu_resources/render_pipeline_pool.rs

```diff
@@ -210,13 +226,7 @@ impl GpuRenderPipelinePool {
                     re_log::info!(label = desc.label.get(), "recompiled render pipeline");
                     Some(sm)
                 }
-                Err(err) => {
-                    re_log::error!(
-                        err = re_error::format(err),
-                        "couldn't recompile render pipeline"
-                    );
-                    None
-                }
+                Err(_) => None,
```

Comment:
Don't ignore errors - either return them or log them

```suggestion
                Err(err) => {
                    re_log::error!("Failed to compile render pipeline: {}", re_error::format(err));
                    None
                }
```

Reply:
I can't seem to find a convenient way to  log the `err` here